app/server/tests/api-deletePostTags.test.js

```diff
@@ -0,0 +1,72 @@
+const { AuthFunctions } = require('./functions/authFunctions');
+const { password, name, email } = require('./functions/randomizer');
+const { PostFunctions } = require('./functions/postFunctions');
+const community = require('./functions/communityFunctions');
+
+let auth = new AuthFunctions();
+let post = new PostFunctions();
+
+
+describe('Testing user\'s ability to DELETE Post Tags', () => {
+    let loginResponse;
+    let response;
+    let postResponse;
+    let userName = name.gen();
+    let userPassword = password.gen();
+    let communityName = name.gen();
+    let randomText = name.gen();
+    let tag1 = 'great';
+
+    beforeAll(async () => {
+        // Set up user
+        await auth.register(userName, email.gen(), userPassword);
+        loginResponse = await auth.login(userName, userPassword);
+
+        // Create some communities
+        await community.createCommunity(communityName, randomText, randomText, [{"tag": tag1, "count": 1}], loginResponse.body.token);
+
+        // Join communities
+        await community.joinCommunity(communityName, loginResponse.body.token);
+
+        // Create some posts
+        postResponse = await post.createPost(randomText, randomText, communityName, loginResponse.body.token);
+
+        // Tag that post
```

Comment:
You can only tag posts with the tags that have been set by the community creator.

Reply:
Looks like you added tags after community creation in another test, but not this one.

Reply:
The community is created in the beforeAll block, so it is all one user here who is registered, logged in, creates a community, joins it, posts, then tags that post before the first test case is even reached. A lot of set up just to test deletion. Tags were added to community during creation here. 

There was a test where I added a second tag to community, but that was case-specific to that test.